managed/cmd/pmm-managed-init/main.go

```diff
@@ -34,6 +35,12 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+	if ok, _ := strconv.ParseBool(os.Getenv("PMM_DISABLE_BUILTIN_POSTGRES")); ok {
+		if err := supervisord.SavePMMConfig(map[string]any{"DisableInternalDB": ok}); err != nil {
+			logrus.Errorf("PMM Server configuration error: %s.", err)
+			os.Exit(1)
+		}
+	}
```

Comment:
we need to reload supervisord after that

Reply:
In general, this is not required there, since pmm-managed-init, judging by [entrypoint.sh](https://github.com/percona/pmm/blob/main/build/docker/server/entrypoint.sh#L25-L30), is started before supervisord.
`# pmm-managed-init validates environment variables.
pmm-managed-init

# Start supervisor in foreground
exec supervisord -n -c /etc/supervisord.conf`